lib/generators/active_admin/resource/templates/admin.rb

```diff
@@ -1,5 +1,5 @@
 ActiveAdmin.register <%= class_name %> do
-<% if Rails::VERSION::MAJOR == 4 || defined?(ActionController::StrongParameters) %>
+<% if Rails::VERSION::MAJOR >= 4 || defined?(ActionController::StrongParameters) %>
```

Comment:
If we have this dependency adapter patter, we should use a something like `ActiveAdmin::Dependency.rails.strong_parameters?`